eth/tracers/logger/logger.go

```diff
@@ -443,6 +444,13 @@ func formatLogs(logs []StructLog) []StructLogRes {
 			}
 			formatted[index].Stack = &stack
 		}
+		if trace.ReturnData != nil {
```

Comment:
I realize you're breaking it into 32-byte words to be consistent with memory output. I personally don't see a strong reason for it. And memory is usually much bigger. So my vote is just to return a single hex string.

Reply:
@holiman is there any benefits to chunking it for the fuzzers/evmlab etc?